static/src/js/zustand/types.ts

```diff
@@ -145,6 +145,18 @@ export type FocusedCollectionSlice = {
   }
 }
 
+export type FocusedGranuleSlice = {
+  /** The Focused Granule Slice of the store */
+  focusedGranule: {
+    /** The currently focused granule */
+    focusedGranule: string | null
+    /** Function to set or remove the focused granule */
+    changeFocusedGranule: (granuleId: string | null) => void
+    /** Function to get the focused granule metadata */
+    getFocusedGranule: () => void
```

Comment:
If this is getting the granule, seems like `void` might not be the right return type?

Reply:
Nothing is returned from the function, another part of the store receives the data, but no actual return

Reply:
:ill_see_myself_out: